plugins/pom-analyzer/src/main/java/eu/f4sten/pomanalyzer/MyInjectorConfig.java

```diff
@@ -15,6 +15,8 @@
  */
```

Comment:
Can we use a better name for this file? e.g. MyInjectorConfig -> PomAnalyzerConfig

Reply:
Yes, this is a fair point, but please keep some points in mind:

- at no point in the code will this class be referenced. It will only be loaded through reflection from the loader.
- we are not configuring the `PomAnalyzer`, but the `Injector` with it... so a proper full name would be `PomAnalyzerInjectorConfig`

With naming, I think there is a tradeoff to make between expressiveness and ease of use. I would actually propose to rename `PomAnalyzer` to `Main`, so the entrypoint is clear for everybody (and also this class won't be referenced anywhere in the code). For the config, I am not so sure what a good name would be... we could even just call it `Config` or `InjectorConfig`. The only *startup class* of a plugin that *might* be relevant for someone else is the `*Args` class, here we should include the defining plugin in its name, like `PomAnalyzerArgs`.

Reply:
Yes that seems ok to me